bee-api/bee-rest-api/src/types/dtos.rs

```diff
@@ -1182,8 +1182,12 @@ pub struct TimelockUnlockConditionDto {
     #[serde(rename = "type")]
     pub kind: u8,
     #[serde(rename = "milestoneIndex")]
+    #[serde(skip_serializing_if = "is_zero_milestone")]
+    #[serde(default)]
```

Comment:
I think this can be like
```suggestion
    #[serde(skip_serializing_if = "is_zero_milestone", default)]
```

Reply:
fixed